services/ui-src/src/types/reports.ts

```diff
@@ -0,0 +1,30 @@
+import { CustomHtmlElement, FormJson } from "types";
+
+export interface ReportPageVerbiage {
+  intro: {
+    section: string;
+    subsection?: string;
+    spreadsheet?: string;
```

Comment:
I don't think we'll need the Spreadsheet icon for MFP, but we'll double-check with R&D!